### PR TITLE
Typo, duplicated dict key, and write to log

### PR DESCRIPTION
**Pull request by kinow**
Hi @rosepearson 

Here's a few updates found after checking out the code. Couldn't run the examples, nor the tests. I tried `pip install .` but it failed due to PDAL. Tried installing the lib-something-pdal on Ubuntu, didn't fix. Tried to create a Conda environment, but it was taking ages and in the end conda crashed :sweat_smile: 

So just read some parts of the code, ran a quick IDE analysis, and prepared this PR fixing some of the IDE warnings.

Cheers
Bruno

### CODE REVIEW

File: src/geofabrics/geometry.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def y(self):
     def z(self):
         """ The sampled contour z values """
         if self._z is None:
-            # map depth to elevatation
```
Comment (rosepearson) [inline src/geofabrics/geometry.py:210]: The changes all look good! Thanks for making them.

**Review by rosepearson — APPROVED** [review]
Thanks for the commits Bruno. The changes look great. I'll look into the other issues you experienced RE easily installing/running the package separately.

**Comment by kinow** [comment]
> Thanks for the commits Bruno. The changes look great. I'll look into the other issues you experienced RE easily installing/running the package separately.

Thanks @rosepearson !

**Comment by rosepearson** [comment]
I will leave this to merge when you're ready. Let me know if you want me to act overwise.

**Pull request merged**